### PR TITLE
Feature/m2 1365  dataviz  feedback  reviewed tab

### PR DESCRIPTION
**Pull request by nmialik**

### CODE REVIEW

File: src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/AssessmentControllers/SingleSelectionController/SingleSelectionController.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { Controller, FieldValues } from 'react-hook-form';
+
+import { SingleSelection } from 'modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/AssessementItems';
```
Comment (kdanilenka32) [inline src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/AssessmentControllers/SingleSelectionController/SingleSelectionController.tsx:3]: maybe better to use ../../ here

File: src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/AssessmentControllers/SliderController/SliderController.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { Controller, FieldValues } from 'react-hook-form';
+
+import { Slider } from 'modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/AssessementItems';
```
Comment (kdanilenka32) [inline src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/AssessmentControllers/SliderController/SliderController.tsx:3]: same here

File: src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/FeedbackAssessment/ActivityCardItemList/ActivityCardItem/ItemCardButtons/ItemCardButtons.tsx [hunk 3]
```diff
@@ -18,13 +18,16 @@ export const ItemCardButtons = ({
   const { t } = useTranslation();
   const { watch } = useFormContext();
 
-  const answerValues = watch(`answers.${step}.answer.value`);
+  const getValue = (value: string | number | string[]) =>
+    !!(Array.isArray(value) ? value.length : value);
```
Comment (kdanilenka32) [inline src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/FeedbackAssessment/ActivityCardItemList/ActivityCardItem/ItemCardButtons/ItemCardButtons.tsx:22]: Do y need !! here?

File: src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/AssessmentControllers/utils.ts
Comment (IhorNesterenko99) [inline src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/AssessmentControllers/utils.ts:1]: could you pls rename to AssesmentControllers.utils.ts?

**Review by IhorNesterenko99 — APPROVED** [review]

**Review by kdanilenka32 — APPROVED** [review]

**Pull request merged**